src/assignment/helpers/duplicate-assignment.ts

```diff
@@ -0,0 +1,38 @@
+import { Avo } from '@viaa/avo2-types';
+import { TFunction } from 'i18next';
+
+import { CustomError } from '../../shared/helpers';
+import { ToastService } from '../../shared/services';
+import { AssignmentService } from '../assignment.service';
+
+export function duplicateAssignment(
+	t: TFunction,
```

Comment:
you can use the global t function as long as you only use it in a function that gets called after the app is booted. Don't use it in consts defined at the root level

```
import i18n from '../../shared/translations/i18n';

function ...() {
  i18n.t('...')
}
```